cram/cram_structs.h

```diff
@@ -192,7 +192,7 @@ typedef struct cram_file_def {
 struct cram_slice;
 
 enum cram_block_method {
-    ERROR    = -1,
```

Comment:
I just noted that the affected header files aren't part of the public API. So perhaps less of an issue than I originally thought.